profiles/kentik_snmp/hpe/hpe-synergy.yml

```diff
@@ -0,0 +1,60 @@
+#https://mibs.observium.org/mib/SYNERGY100G-HPE-CPU-UTIL-MIB/
+#https://mibs.observium.org/mib/SYNERGY100G-HPE-NLP-MIB/#syn100GhpeNLPModuleConfig
+---
+extends:
+  - system-mib.yml
+  - if-mib.yml
+  - ip-mib.yml
+
+sysobjectid:
+  - 1.3.6.1.2.1.11.5.7.5.9    # VC SE 100Gb F32 Module
+
+
+provider: kentik-server-chassis
+
+metrics:
+  - MIB: HPE-SYNERGY-MIB
+    symbols:
+      - name:  syn100GhpeSwitchMaxCPUThreshold
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.1.1
+        tag: MAX_CPU
+        poll_time_sec: 60
+      - name:  syn100GhpeSwitchMinCPUThreshold
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.1.2
+        tag: MIN_CPU
+        poll_time_sec: 60
+      - name:  syn100GhpeSwitchAverageCPUUtilization  
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.2.1
+        tag: AVG_CPU
+        poll_time_sec: 60
+  - MIB: HPE-SYNERGY-MIB
+    table:
+      name: syn100GhpeNLPServerPortTable
+      OID: 1.3.6.1.4.1.11.5.7.5.9.1.4060.5
+      
+    symbols:
+      - name: syn100GhpeNLPServerPortEntry
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4060.5.1
+      - name: syn100GhpeNLPServerPortNumber
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4060.5.1.1 
+      - name: syn100GhpeNLPServerPortResetLoopDetection 
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4060.5.1.2
+        enum:
+          true: 1
+          false: 2
+  - MIB: HPE-SYNERGY-MIB
+    table:
+      name: syn100GhpeNLPServerPortStatsTable
+      OID: 1.3.6.1.4.1.11.5.7.5.9.1.4060.6
+      
+    symbols:
+      - name: syn100GhpeNLPServerPortStatsEntry 
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4060.6.1
+      - name: syn100GhpeNLPStatsPortNumber
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4060.6.1.1 
+      - name: syn100GhpeNLPLoopDetectedStatus 
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4060.6.1.2
```

Comment:
I cannot find this MIB on the public internet, but I am relatively certain this OID needs to have an `enum` section to map the numeric value returned with an actual status name in a string. If you have access to the vendor's MIB, it will show the definition pretty clearly. (You can upload the MIB here and I can verify if you need help)